<launch>
    <arg name="simulation" value="true"/>
    <arg name="grasps_num_per_object" value="2"/>
    <!-- <arg name="record_data_path" value="/mnt/data_sal/data_kai/multi_finger_grasp_data/"/> -->
    <arg name="record_data_path" value="/media/kai/multi_finger_grasp_data/"/>
    <arg name="camera_information_topic" value="/kinect2/hd/camera_info"/>
    <arg name="use_hd_arg" value="true"/>
    <arg name="save_urdf_path_arg" value="/media/kai/sim_dataset/BigBird/BigBird_urdf"/>
    <arg name="object_meshes_path_arg" value="/media/kai/sim_dataset/BigBird/BigBird_mesh"/>

   <!-- <node name="manage_gazebo_scene_node" pkg="grasp_pipeline" type="manage_gazebo_scene_server.py" output="screen" >
      <param name="save_urdf_path" value="$(arg save_urdf_path_arg)" type="string"/>
      <param name="object_meshes_path" value="$(arg object_meshes_path_arg)" type="string"/>
    </node> -->

    <group if="$(arg simulation)">
	<!--pass the path to blend file as an argument-->
    <node name="blensor_ros" pkg="blensor_ros" type="blensor.sh" output="screen" args="$(find blensor_ros)/extras/default.blend false"/>
    <!--<node name="robot_state_srv" pkg="blensor_ros" type="robot_state_srv.py" output="screen"/> -->
    </group>
   
</launch>
